src/Analyzers/Analyzers/src/UseParameterAttributeCorrectlyAnalyzer.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.AspNetCore.Analyzers.Helpers;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Microsoft.AspNetCore.Analyzers
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
+    public sealed class UseParameterAttributeCorrectlyAnalyzer : DiagnosticAnalyzer
+    {
+        private static readonly DiagnosticDescriptor s_rule = new(
+#pragma warning disable RS2008 // Enable analyzer release tracking
```

Comment:
I didn't enable release tracking as I see this is suppressed in other analyzers as well.